docs/api/spec/query-api.yaml

```diff
@@ -0,0 +1,306 @@
+      #   content:
+      #     text/plain: 
+      #       example: '{{ACCESS_TOKEN}}'
+      #   required: true
+      responses:
+        '200':
+          description: 'Successfully like the object response'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/likableObject'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '422':
+          description: 'Missing request parameters'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/likableObject'
+              example:
+                code: 422
+                message: 'object_type is not allowed to be empty'
+      deprecated: false
+  '/info/types/{type_id}/things/{object_id}':
+    get:
+      tags:
+        - Get info on a likable object
+      summary: Get info on a likable object
+      description: 'Get info on a likable object'
+      parameters:
+        - name: type_id
+          description: 'ID of specific token to retrieve'
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 1
+        - name: object_id
+          description: 'ID of specific token to retrieve'
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 186734
+      responses:
+        '200':
+          description: 'Successfully like the object response'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/likableObject'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '422':
+          description: 'Missing request parameters'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/likableObject'
+              example:
+                code: 422
+                message: 'object_type is not allowed to be empty'
+      deprecated: false
+  '/types':
+    get:
+      tags:
+        - Types info and manipulation
+      summary: Get types name and id
+      description: 'Get types name and id'
+      responses:
+        '200':
+          description: 'Successfully retrieve types info'
+          content:
+            application/json:
+              example:
+                1: countries
+                2: organizations
+                3: planters
+                4: species
+                5: trees
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+      deprecated: false
+    post:
+      tags:
+        - Types info and manipulation
+      summary: Create a new type in the database
+      description: 'Create a new type in the database'
+      requestBody:
+        description: 'Name of new type'
+        content:
+          text/plain: 
+            example: 'trees'
```

Comment:
Please always use application/json (be consistent) to communicate, so here could be:
```
{
  name: 'trees'
}
```
also this make it extensible.